hack/test-cmd.sh

```diff
@@ -55,27 +58,45 @@ unset KUBECONFIG
 
 export ALLOWED_REGISTRIES='[{"domainName":"172.30.30.30:5000"},{"domainName":"myregistry.com"},{"domainName":"registry.centos.org"},{"domainName":"docker.io"},{"domainName":"gcr.io"},{"domainName":"quay.io"},{"domainName":"*.redhat.com"},{"domainName":"*.docker.io"},{"domainName":"registry.redhat.io"}]'
 
-os::start::configure_server
+eval $( ./hack/install-etcd.sh | tail -n1 )
```

Comment:
This immediately stands out as non-standard and potentially dangerous. Why?

Reply:
Updated this to explicitly re-export PATH based on the string returned by the `./hack/install-etcd.sh` script